docs/index.md

```diff
@@ -8,17 +8,54 @@
 <p>A Text-to-Speech Transformer in TensorFlow 2</p>
 </h2>
 
+<p class="text">All samples are converted with <a href="https://github.com/fatchord/WaveRNN"> WaveRNN </a>vocoder.</p>
 
-### 🎧 Samples from the autoregressive model, converted with [WaveRNN](https://github.com/fatchord/WaveRNN) vocoder
+## 🎧 Forward model samples
```

Comment:
I would not split the forward and autoregressive part instead doing something like this:

```
## 🎧 Model samples

<p class="text">President Trump met with other leaders at the Group of twenty conference.</p>

| forward | autoregressive |
|:---:|:---:|
|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/trump.wav?raw=true" controls preload></audio>|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/Trump.wav?raw=true" controls preload></audio>|

<p class="text">Scientists, at the CERN laboratory, say they have discovered a new particle.</p>

| forward | autoregressive |
|:---:|:---:|
|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/scientists.wav?raw=true" controls preload></audio>|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/cern_particle.wav?raw=true" controls preload></audio>|

<p class="text">There’s a way to measure the acute emotional intelligence that has never gone out of style.</p>

| forward | autoregressive |
|:---:|:---:|
|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/EQ.wav?raw=true" controls preload></audio>|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/EQ.wav?raw=true" controls preload></audio>|

<p class="text">The Senate's bill to repeal and replace the Affordable Care-Act is now imperiled.</p>

| forward | autoregressive |
|:---:|:---:|
|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_forward_transformer/senate.wav?raw=true" controls preload></audio>|<audio src="https://github.com/as-ideas/tts_model_outputs/blob/master/ljspeech_transformertts/affordablecareact.wav?raw=true" controls preload></audio>|
```

Reply:
Much better